CMakeLists.txt

```diff
@@ -34,7 +34,11 @@ if(BINARY_RELEASE)
   set(ASAN OFF)
   set(UBSAN OFF)
   set(DEBUG OFF)
-  set(LTO ON)
+    if(AMIGA)
+      set(LTO OFF)

```

Comment:
Perhaps LTO check should be checked below all the different platforms so we can disable LTO on platforms that it's known not to work on?

Reply:
Here if(NOT WARPOS) works.